src/neo/Ledger/Blockchain.cs

```diff
@@ -350,7 +305,7 @@ protected override void OnReceive(object message)
                     OnPreverifyCompleted(task);
                     break;
                 case Idle _:
-                    if (MemPool.ReVerifyTopUnverifiedTransactionsIfNeeded(MaxTxToReverifyPerIdle, View))
+                    if (system.MemPool.ReVerifyTopUnverifiedTransactionsIfNeeded(MaxTxToReverifyPerIdle, system.StoreView))
```

Comment:
`MemPool` could be moved to other level instead of system.

Reply:
I think `NeoSystem` is the best place.